### PR TITLE
enh: add double quotes before paths so spaced paths can work

### PR DESCRIPTION
**Pull request by vkt1414**
Currently whenever a download path consists of spaces, s5cmd command does not work. 

image

But to force it, we must include the path containing spaces in double quotes. The issue was first encountered in SlicerIDCBrowser. https://github.com/ImagingDataCommons/SlicerIDCBrowser/issues/39

This PR will fix https://github.com/ImagingDataCommons/SlicerIDCBrowser/issues/39

### CODE REVIEW

**Comment by fedorov** [comment]
Of course, this makes sense - we did the same for the portal, but I didn't connect the dots...

image

File: idc_index/index.py [hunk 1]
```diff
@@ -918,15 +928,17 @@ def _parse_s5cmd_sync_output_and_generate_synced_manifest(
...
                 list_of_directories = synced_df.path.to_list()
             else:
                 synced_df["s5cmd_cmd"] = (
-                    "sync " + synced_df["s3_url"] + " " + downloadDir
+                    "sync " + synced_df["s3_url"] + " " + '"' + downloadDir + '"'
                 )
                 list_of_directories = [downloadDir]
-            synced_df["s5cmd_cmd"].to_csv(synced_manifest, header=False, index=False)
+            synced_df["s5cmd_cmd"].to_csv(
+                synced_manifest, header=False, index=False, quoting=csv.QUOTE_NONE
```
Comment (fedorov) [inline idc_index/index.py:940]: Can you explain the motivation and use of the `quoting` parameter?

Comment (vkt1414) [reply idc_index/index.py:940]: I had to dig deep for this one. When I add double quotes to the path and write to csv, csv contained another set of double quotes for each line.
something like "cp s3://idc-open-data/2629b726-1d5e-46bb-ad83-84e64c23309b/* "test"". 

 Apparently this is normal while writing to csv unless we disable with csv.QUOTE_NONE

Comment (fedorov) [reply idc_index/index.py:940]: I read the commentary here https://www.perplexity.ai/search/explain-the-consequences-7kbYdW0xTUaERrsA3zj8AQ, and it comes with a lot of warnings. I need to think more about this...

Comment (fedorov) [reply idc_index/index.py:940]: The thing is we *may* have strange characters in our table - DICOM `SeriesDescription` may as well have garbage..

Comment (vkt1414) [reply idc_index/index.py:940]: The current hierarchy does not include SeriesDescription.

I do not expect quotes in any of these
collection_id
PatientID
StudyInstanceUID
Modality
SeriesInstanceUID"

Comment (vkt1414) [reply idc_index/index.py]: bing suggested another simpler that does not need csv QUOTENone.

Comment (fedorov) [reply idc_index/index.py]: You are right, I somehow thought about all of the columns.

**Review by fedorov — APPROVED** [review]

**Pull request merged**